mmseg/datasets/transforms/loading.py

```diff
@@ -493,3 +494,136 @@ def transform(self, single_input: Union[str, np.ndarray, dict]) -> dict:
         if 'img' in inputs:
             return self.from_ndarray(inputs)
         return self.from_file(inputs)
+
+
+@TRANSFORMS.register_module()
+class LoadSingleRSImageFromFile(BaseTransform):
+    """Load a Remote Sensing mage from file.
+
+    Required Keys:
+
+    - img_path
+
+    Modified Keys:
+
+    - img
+    - img_shape
+    - ori_shape
+
+    Args:
+        imdecode_backend (str): The image decoding backend type.
+            The image backend decoding type is switched from cv2 to GDAL.
+            Notes: GDAL>=3.4.2 required.
+        to_float32 (bool): Whether to convert the loaded image to a float32
+            numpy array. If set to False, the loaded image is a float64 array.
+            Defaults to True.
+    """
+
+    def __init__(self,
+                 imdecode_backend: str = 'gdal',
+                 to_float32: bool = True):
+        self.imdecode_backend = imdecode_backend
```

Comment:
The `imdecode_backend` seems not been used, it's a parameter for [LoadImageFromFile](https://github.com/open-mmlab/mmcv/blob/main/mmcv/transforms/loading.py#L14), but not `BaseTransform`.